configure.ac

```diff
@@ -293,6 +293,7 @@ if test "$enable_man" = "yes"; then
 fi
 AM_CONDITIONAL(ENABLE_REGENERATE_MAN, test "x$enable_man" != "xno")
 
+AC_CHECK_HEADERS(sys/prctl.h, [], [enable_subids="no"])
```

Comment:
Please quote everything:

```
AC_CHECK_HEADERS([sys/prctl.h], [], [enable_subids="no"])
```

We recently applied a commit to quote everything consistently:
<https://github.com/shadow-maint/shadow/commit/630fb7ece1f731cb7ba85064d3568c9e130c9fbd>